container/shim/src/modules/registration.js

```diff
@@ -193,6 +193,16 @@ async function checkCertValidity(certBuffer, registerOptions) {
     valid = false;
   }
 
+  if (
+    NETWORK === "main" &&
+    cert.subjectAltName &&
+    !cert.subjectAltName.includes(".l1s.saturn.ms") &&
```

Comment:
comment here would be nice to describe intent.

If I read right, this is:
- you have a cert and your cert is not unique to your node (i.e. new naming scheming .l1s.saturn.ms)
- your roll a 1 on a D1000 dice :)
- then we mark the cert invalid and get a new one.

And this runs every 8 min on every node? So that means for 3000 nodes we're doing about 3 nodes every 8 minutes -- is that our target?

Reply:
Yes for most. The validity check is only on initial registration or every 24 hours, so 3 nodes every update / day